resources/views/home.blade.php

```diff
@@ -0,0 +1,76 @@
+@extends('layouts.app')
```

Comment:
this home.blade.php is not implemented yet so you can remove this for now

Reply:
Fixes issue(Delete)

"Thank you comments. Aim for simple code."